src/components/Confirm/index.js

```diff
@@ -83,35 +81,11 @@ const Previews = ({capture, retakeAction, confirmAction} ) =>
   </div>
 
 
-export default class Confirm extends Component {
-  completeCapture(method, side, capture, confirmCapture, nextStep) {
-    confirmCapture({method, id: capture.id})
-    confirmEvent(method, side)
-    nextStep()
-  }
-
-  startApiUpload(method, side, capture, token, confirmCapture, nextStep, advancedValidation, onApiUpload, onApiError) {
-    onApiUpload()
-    postToOnfido(capture, method, token, advancedValidation, () => this.completeCapture(method, side, capture, confirmCapture, nextStep), onApiError)
-  }
-
-  render({nextStep, method, side, validCaptures, token, onApiUpload, onApiError,
-    advancedValidation, actions: {deleteCaptures, confirmCapture}}) {
-      const capture = validCaptures[0]
-      return (
-        <Previews
-          capture={capture}
-          retakeAction={() => deleteCaptures({method, side})}
-          confirmAction={() => this.startApiUpload(method, side, capture, token, confirmCapture, nextStep, advancedValidation, onApiUpload, onApiError) }
-        />
-      )
-    }
-}
+const Confirm = ({method, side, validCaptures:[capture], onConfirm, actions: {deleteCaptures}}) =>
```

Comment:
not truly necessary. It's just that the good think about `Previews` is that it's truly a presentation only component. Whereas `Confirm` produces side effects.